plugins/available/base.plugin.bash

```diff
@@ -39,20 +39,27 @@ function pickfrom ()
     head -n $n $file | tail -1
 }
 
-function pass ()
+function passgen ()
 {
     about 'generates random password from dictionary words'
     param 'optional integer length'
     param 'if unset, defaults to 4'
-    example '$ pass'
-    example '$ pass 6'
+    example '$ passgen'
+    example '$ passgen 6'
     group 'base'
-    local i pass length=${1:-4}
+    local i passgen length=${1:-4}
     pass=$(echo $(for i in $(eval echo "{1..$length}"); do pickfrom /usr/share/dict/words; done))
```

Comment:
Shouldn't this be `passgen` instead of `pass`? Same for the next couple of lines.